### PR TITLE
Added a simple mission

### PR DESCRIPTION
**Pull request by ghost**
In this mission, the player has to open the door.
The mission starts within GameMode and finishes when the door is triggered.
#28 
desktop screenshot 2019 02 24 - 19 14 20 74

### CODE REVIEW

File: Config/DefaultEngine.ini [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [/Script/EngineSettings.GameMapsSettings]
 GameDefaultMap=/Game/ThirdPersonCPP/Maps/inner10range.inner10range
-EditorStartupMap=/Game/ThirdPersonCPP/Maps/inner10range.inner10range
+EditorStartupMap=/Game/ThirdPersonCPP/Maps/MissionMap.MissionMap
```
Comment (UnoYakshi) [inline Config/DefaultEngine.ini]: No need.

File: Config/DefaultEngine.ini [hunk 2]
```diff
@@ -25,51 +25,6 @@ AppliedDefaultGraphicsPerformance=Maximum
...
-bSubstepping=False
-bSubsteppingAsync=False
-MaxSubstepDeltaTime=0.016667
-MaxSubsteps=6
-SyncSceneSmoothingFactor=0.000000
-AsyncSceneSmoothingFactor=0.990000
-InitialAverageFrameRate=0.016667
-PhysXTreeRebuildRate=10
-DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,MBPBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPNumSubdivs=2)
-
```
Comment (UnoYakshi) [inline Config/DefaultEngine.ini:72]: Why have you changed the settings so much?..

File: Config/DefaultEngine.ini [hunk 3]
```diff
@@ -126,3 +81,48 @@ DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,MBPBounds
...
+MaxPhysicsDeltaTime=0.033333
+bSubstepping=False
+bSubsteppingAsync=False
+MaxSubstepDeltaTime=0.016667
+MaxSubsteps=6
+SyncSceneSmoothingFactor=0.000000
+AsyncSceneSmoothingFactor=0.990000
+InitialAverageFrameRate=0.016667
+PhysXTreeRebuildRate=10
+DefaultBroadphaseSettings=(bUseMBPOnClient=False,bUseMBPOnServer=False,MBPBounds=(Min=(X=0.000000,Y=0.000000,Z=0.000000),Max=(X=0.000000,Y=0.000000,Z=0.000000),IsValid=0),MBPNumSubdivs=2)
```
Comment (UnoYakshi) [inline Config/DefaultEngine.ini:128]: Not sure why and what was that...

**Comment by ghost** [comment]
Added 2 enemies to kill.
desktop screenshot 2019 03 03 - 03 35 37 00

**Review by UnoYakshi — APPROVED** [review]

**Pull request merged**